static/js/components/NotificationContainer.js

```diff
@@ -32,6 +32,15 @@ export class NotificationContainer extends React.Component<Props, State> {
     const { removeUserNotification, messageRemoveDelayMs } = this.props
     const { hiddenNotifications } = this.state
 
+    if (notificationKey === "cms-site-wide-notification") {
```

Comment:
1. Please create a constant for this value (`"cms-site-wide-notification"`) and use it here and in App.js
1. Can you include this comment above the `if` statement?
    
    ```javascript
    // If the notification came from the CMS, set a local storage value to
    // avoid showing again.
    ```